deltachat-contact-tools/src/lib.rs

```diff
@@ -44,14 +44,25 @@ use regex::Regex;
 pub struct VcardContact {
     /// The email address, vcard property `email`
     pub addr: String,
-    /// The contact's display name, vcard property `fn`
-    pub display_name: String,
+    /// Name authorized by the contact itself, vcard property `fn`. Can be empty, one should use
+    /// `display_name()` to obtain the display name.
+    pub authname: String,
```

Comment:
Just a note, nothing blocking: It's not actually "authorized by the contact itself", because vCards are sent by a third party (even though AFAIK it's true that the `authname` in `contact.rs` has the same semantics). And the code handling `authname`s in `contact.rs` is a bit far away, while this crate here is only concerned with parsing vCards.

Since i agree that `display_name` is something else in DeltaChat land, I would slightly be in favor of calling this field simply `name`, but feel free to ignore this comment.

Reply:
I'd avoid using `name`/`display_name` to protect from occasional sharing of locally given names. With `authname` the code in the deltachat core looks just straightforward. But `public_name`/`shared_name` would be also ok. Anyway i agree that the comment should be fixed